data/events.json

```diff
@@ -849,37 +849,46 @@
 	  "id": 136,
 	  "name": "Feathers",
 	  "description": "\"Feathers\", is the annual creative writing event conducted during Effervescence.",
-	  "location": "CC3",
+	  "location": "Online",
 	  "timestamp": 2,
 	  "imageUrl": "https://github.com/Effervescence-IIITA/Effervescence18/raw/master/images/events/feathers.jpg",
-	  "facebookEventLink": "",
+	  "facebookEventLink": "https://www.facebook.com/sarasva.litclub/",
 	  "additionalInfo": [],
 	  "categories": [
 		"literature"
 	  ],
-	  "organizers": []
+	  "organizers": [
+		  {
+		  "name": "Srishti Upadhyay",
+		  "phoneNumber" :9453033948
+		  }
+	  ]
 	},
 	{
 	  "id": 137,
 	  "name": "Cognoscentia",
 	  "description": "Congnoscentia is organised each year to put your grey cells into work. This mind numbing quizzing competition covers every genre, Literally, everything under the sun!",
 	  "location": "CC3",
 	  "timestamp": 1508065200,
-	  "imageUrl": "https://github.com/Effervescence-IIITA/Effervescence18/raw/master/images/events/cognoscentia.jpg",
-	  "facebookEventLink": "https://www.facebook.com/events/141002776519860/?ref=br_rs",
+	  "imageUrl": "https://scontent-bom1-1.xx.fbcdn.net/v/t1.0-9/42730855_2188101164593150_4025136564039319552_o.jpg?_nc_cat=104&oh=3ad65df32d5da5f045c03d0c74c13522&oe=5C28AB18",
```

Comment:
add the image according to the URL specified. Also, upload the respective image in events/<eventname>.jpg.